src/app/problem/right/Description.js

```diff
@@ -0,0 +1,57 @@
+import Block from "./Block";
+import {useState} from "react";
+import MarkdownEditor from "../commons/MarkdownEditor";
+import NewMarkdownEditorWriteTab from "../commons/NewMarkdownEditorWriteTab";
+import NewMarkdownEditorPreviewTab from "../commons/NewMarkdownEditorPreviewTab";
+import {EditorContext} from "../commons/NewMarkdownEditorContext";
```

Comment:
Rename 
- `NewMarkdownEditorWriteTab` to `MarkdownEditorWriteTab`.
- `NewMarkdownEditorPreviewTab` to `MarkdownEditorPreviewTab`.
- `NewMarkdownEditorContext` to `MarkdownEditorContext`.